### ISSUE TITLE
Tensorflow 2.0.0 / tf.keras.layers.TimeDistributed layer can't be save to saved Model

### ISSUE THREAD

**Issue by luvwinnie** [tensorflow/tensorflow#33094]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution : Colaboratory (GPU Runtime)
- tensorflow version: 2.0.0
- python version: 3.6.8

**Describe the current behavior**
The model defined which has tf.keras.layers.timeDistributed layer cannot be save by model.save() function.

It shows the error below

```
ValueError                                Traceback (most recent call last)
/usr/lib/python3.6/inspect.py in getfullargspec(func)
   1125                                        skip_bound_arg=False,
-> 1126                                        sigcls=Signature)
   1127     except Exception as ex:

41 frames
ValueError: no signature found for builtin <tensorflow.python.keras.saving.saved_model.save_impl.LayerCall object at 0x7f74467284a8>

The above exception was the direct cause of the following exception:

TypeError                                 Traceback (most recent call last)
/usr/lib/python3.6/inspect.py in getfullargspec(func)
   1130         # else. So to be fully backwards compatible, we catch all
   1131         # possible exceptions here, and reraise a TypeError.
-> 1132         raise TypeError('unsupported callable') from ex
   1133 
   1134     args = []

TypeError: unsupported callable
```

**Describe the expected behavior**
In tensorflow 2.0.0 supposed the model.save model default to be saved in SavedModel format. 

**Code to reproduce the issue**
```mirrored_strategy = tf.distribute.MirroredStrategy()

def get_data():
  datasets, ds_info = tfds.load(name='mnist', with_info=True, as_supervised=True)
  mnist_train, mnist_test = datasets['train'], datasets['test']

  BUFFER_SIZE = 10000

  BATCH_SIZE_PER_REPLICA = 64
  BATCH_SIZE = BATCH_SIZE_PER_REPLICA * mirrored_strategy.num_replicas_in_sync

  def scale(image, label):
    image = tf.cast(image, tf.float32)
    image /= 255

    return image, label

  train_dataset = mnist_train.map(scale).cache().shuffle(BUFFER_SIZE).batch(BATCH_SIZE)
  eval_dataset = mnist_test.map(scale).batch(BATCH_SIZE)

  return train_dataset, eval_dataset

def get_model():
  with mirrored_strategy.scope():
    model = tf.keras.Sequential([
        tf.keras.layers.Conv2D(32, 3, activation='relu', input_shape=(28, 28, 1)),
        tf.keras.layers.MaxPooling2D(),
        # tf.keras.layers.Flatten(),
        tf.keras.layers.TimeDistributed(tf.keras.layers.Dense(64, activation='softmax')),
        # tf.keras.layers.TimeDistributed(tf.keras.layers.Dense(64, activation='softmax')),
        # tf.keras.layers.Dense(10, activation='softmax')
    ])

    model.compile(loss='sparse_categorical_crossentropy',
                  optimizer=tf.keras.optimizers.Adam(),
                  metrics=['accuracy'])
    return model

model = get_model()
model.save("test_save")
```

**Other info / logs**
The not only can be reproduce in Colaboratory, but also in normal Ubuntu machien which installed with tensorflow-gpu 2.0.0

**Comment by rmothukuru** [comment]
Could reproduce this issue with TF Version 2.0. Here is the [Gist](https://colab.sandbox.google.com/gist/rmothukuru/c698628e49bce4d456daac7c2ddf3bea/33094.ipynb).

**Comment by aldenks** [comment]
Also running into this. Here's a concise test case:
```
model = tf.keras.Sequential([tf.keras.layers.TimeDistributed(tf.keras.layers.Dense(1), input_shape=(None, 1))])
model.save('test_save')
```

**Comment by luvwinnie** [comment]
does this issues got any progress?

**Comment by SavanK** [comment]
Any update?

**Comment by k-w-w** [comment]
This appears to be an error with Python 3. Submitting a fix to resolve this.

**Comment by luvwinnie** [comment]
so is this been fixed to tensorflow-gpu 2.0 stable build? or i have to install another build?
